clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/Keys.groovy

```diff
@@ -87,6 +89,25 @@ class Keys {
               result << [id: parts[2], account: parts[3], region: parts[4]]
             }
             break
+          case Namespace.IMAGES.ns:
+            if (parts.length == 4)
+              result << [account: parts[2], imageId: parts[3]]
+            break
+          case Namespace.SERVER_GROUPS.ns:
+            def names = Names.parseName(parts[5])
+            if (parts.length == 6) {
+              result << [
+                application: names.app.toLowerCase(),
```

Comment:
ha ... when reformatting this moved it into multiple lines.  Since it has 6 parts, I think it is more readable this way.